gradle-enterprise-conventions-maven-extension/extension/.mvn/extensions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
+    <extension>
```

Comment:
Is there a reason we can't dogfood the conventions extension here, using a previously released version?
There would be a bootstrap issue, where no GE connectivity would be available until the first version is released. But it might clarify how these GE / CCUD plugin versions are different from the ones configured in the extension itself.